Expecto/Expecto.fs

```diff
@@ -966,9 +998,10 @@ module Impl =
         List.length tests |> rand.Next |> List.nth tests
 
       let finishTimestamp =
-        let now = Stopwatch.GetTimestamp()
-        let stressSeconds = config.stress.Value.TotalSeconds
-        now + int64(stressSeconds * float Stopwatch.Frequency)
+        lazy
```

Comment:
It just makes the effective start time of these tests when the first test is started. Was more to do with trying to tweak the stress tests to work well so could come out.